cmd/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -43,14 +43,19 @@ import (
 	"k8s.io/kubernetes/cmd/kube-aggregator/pkg/apis/apiregistration/v1alpha1"
 )
 
-const defaultEtcdPathPrefix = "/registry/kubernetes.io/kube-aggregator"
+const (
+	defaultEtcdPathPrefix = "/registry/kubernetes.io/kube-aggregator"
```

Comment:
apologies for the drive-by, but I'd recommend changing the default to not start with registry--that way there's less chance of it and the main apiserver fighting if they both happen to get pointed at the same etcd. Maybe a different PR. @deads2k